### PR TITLE
Add more Profile Icons and Links to Page Bottoms

### PR DESCRIPTION
**Pull request by kumuditha-udayanga**
# Purpose 
The purpose of this PR is to fix issue #582 

# Approach
- Add custom SVG files for missing font-awesome logos.
- Add Docker and Crunchbase links and logos to the footer of relevant pages.

# Preview
https://kumuditha-udayanga.github.io/fossasia.org

# Screenshot

Screenshot 2019-12-19 at 18 15 57

### CODE REVIEW

**Review by liveHarshit — CHANGES_REQUESTED** [review]
Open links with a new tab

**Comment by kumuditha-udayanga** [comment]
@liveHarshit Changes are done :)

**Review by liveHarshit — APPROVED** [review]

File: img/crunchbase.svg [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+		}
+		.icon-body{
+			transition: all .5s;
+			cursor: pointer;
+		}
+		]]>
+    </style>
+    <g class="icon-body">
+        <path d="M1.34,49.81q0-18.27,0-36.55C1.35,6,6.39.91,13.6.91q35.78,0,71.55,0c7.25,0,12.48,4.91,12.5,12.17q.15,36.78,0,73.54c0,7.12-5.32,12-12.43,12H13.45A11.77,11.77,0,0,1,1.35,86.58Q1.32,68.19,1.34,49.81ZM61.1,36.47V33.75c0-5.44,0-10.87-.05-16.3,0-.65-.66-1.82-1.1-1.86-2.15-.2-4.34-.09-6.51-.09V70.17c2.83-.85,5.92,1.32,8.24-1.58a18.65,18.65,0,0,0,23.65-3.14,19.17,19.17,0,0,0,2-23.38,18.27,18.27,0,0,0-9.48-7.49C72.1,32.56,66.62,33.29,61.1,36.47ZM45.26,57c-2,0-3.95.09-5.9,0a3.51,3.51,0,0,0-3.57,1.79,10.49,10.49,0,0,1-19-6.4,10.51,10.51,0,0,1,19.46-5.75c.37.56.95,1.35,1.46,1.38,2.51.14,5,.06,7.61.06a18.36,18.36,0,0,0-19.44-14.4c-10.36,1.06-17.55,9.18-17.34,19s7.75,18.15,17.15,18.76S43.8,65.64,45.26,57Z"/>
+        <path d="M71.47,63c-6.16-.08-10.59-4.35-10.58-10.2,0-6.14,4.65-11,10.5-10.92,6.17,0,10.59,4.91,10.52,11.56C81.86,58.81,77.2,63.07,71.47,63Z"/>
```
Comment (mariobehling) [inline img/crunchbase.svg]: what is this? While most browsers could show svgs, they dont always look the same everywhere. So, please stay with the existing way to display icons.

Comment (mariobehling) [reply img/crunchbase.svg]: Or am I wrong? @liveHarshit as you accepted this. Should we try?

**Comment by mariobehling** [comment]
Please resolve conflicts in any case. Thanks

**Pull request closed without merging**